swatch-product-configuration/src/main/resources/subscription_configs/RHEL/RHEL_for_x86_EUS.yaml

```diff
@@ -0,0 +1,18 @@
+---
+platform: RHEL
+
+id: rhel-for-x86-eus
+
+#includedSubscriptions:
+#  - rhel-for-x86
```

Comment:
Can you add a TODO comment with the JIRA number where this section becomes uncommented?